Description: Option '-a' added to grep.
Author: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2016-04-16
Index: bogofilter-1.2.4+dfsg1/src/tests/t.dump.load
===================================================================
--- bogofilter-1.2.4+dfsg1.orig/src/tests/t.dump.load
+++ bogofilter-1.2.4+dfsg1/src/tests/t.dump.load
@@ -31,13 +31,13 @@ $BOGOUTIL -C $LEN -d "$DATA" -y 20020815
 #update some words, including some old dates
 $BOGOUTIL -C $LEN -l "$DATA" -y 20021215 < "$srcdir"/inputs/$BASE.upd
 #dump database
-$BOGOUTIL -C $LEN -d "$DATA" | grep -v "^.ENCODING" | $SORT > "$TMPDIR"/dump.load-2.txt
+$BOGOUTIL -C $LEN -d "$DATA" | grep -av "^.ENCODING" | $SORT > "$TMPDIR"/dump.load-2.txt
 
 #confirm that updated words have different counts and dates
 ( diff "$TMPDIR"/dump.load-1.txt "$TMPDIR"/dump.load-2.txt ; echo "" ) | $GREP | tee "$TMPDIR"/diff.1.2.txt >> "$OUT"
 
 #dump wordlist, excluding oldest entries
-$BOGOUTIL -C $LEN -a 20020815 -d "$DATA" | grep -v "^.ENCODING" | $SORT > "$TMPDIR"/dump.load-3.txt
+$BOGOUTIL -C $LEN -a 20020815 -d "$DATA" | grep -av "^.ENCODING" | $SORT > "$TMPDIR"/dump.load-3.txt
 
 #confirm exclusion of oldest entries
 ( diff "$TMPDIR"/dump.load-2.txt "$TMPDIR"/dump.load-3.txt ; echo "" ) | $GREP | tee "$TMPDIR"/diff.2.3.txt >> "$OUT"
